pubs/collab-patterns/collab.Rmd

```diff
@@ -197,4 +198,66 @@ nonsys_authors %>%
 ```
 
 
+```{r collaborator df creation, eval=FALSE}
```

Comment:
You can move this to the setup block

Reply:
but we don't want this to be evaluated every time, and the setup block does get evaluated each time